blockchain/blockchain.go

```diff
@@ -45,6 +49,14 @@ var (
 		},
 		[]string{"type"},
 	)
+	fileSizeMtc = prometheus.NewGaugeVec(
```

Comment:
Three ways to monitor the file size:
1. Export the metrics directly from the cloud service.
2. Run a daemon service or process to report the metrics.
3. Monitor the metrics in db_bolt.go.

The last one is the least preferred if the first two can't work.

Reply:
for 1, it should only be able to monitor at the mount level and cannot monitor specific files.
for 2, it may be a heavy changes for this small requirement.
I think 3 is a good approach that can also address the monitoring of chain-000000xx.db files mentioned above.

Reply:
for 2, there have been existing tools to use, like https://github.com/prometheus/node_exporter